firmware/hw_layer/drivers/gpio/tle6240.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * tle6240.h
+ *
+ * TLE6240GP Smart 16-Channel Low-Side Switch
+ *
+ * @date Dec 29, 2018
+ * @author Andrey Belomutskiy, (c) 2012-2018
```

Comment:
"drivers/gpio" folder idea makes total sense. Just moving tle6240 file into it and new .mk file could be another small pull request in order to move forward